### DISCUSSION TITLE
Can you navigate to a page you just created in Next?

### DISCUSSION THREAD

**Discussion by nonoumasy** [vercel/next.js#33336] [Help]
Is it possible to navigate to a page you just created in Next JS? 
For example, on my site: 
https://trip-journals-web.vercel.app/ 
...a User can create a Travel journal. As soon as they create the journal, I would like them to navigate to an "Add entry page" so they can add entries to that journal. The entries is a sub collection of journals.

**Comment by kylemh** [comment]
This got sorted in the Discord server: https://discord.com/channels/752553802359505017/752668543891276009/931861782660775986
